src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+    private String letterA;
```

Comment:
What is `letterA`?

Reply:
not the best name for 'String'-variable, but from of the regular expression, it is correct https://regex101.com/r/sJ9gM7/796